tests/functional/pod_and_daemons/test_mds_cache_trim_standby.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    brown_squad,
+    skipif_ocs_version,
+    bugzilla,
+)
+from ocs_ci.framework.testlib import E2ETest, tier2, skipif_external_mode
+from ocs_ci.helpers.e2e_helpers import run_metadata_io_with_cephfs
+from ocs_ci.ocs import cluster
+from ocs_ci.ocs.resources.pod import get_pod_logs
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2141422")
+@brown_squad
+@skipif_ocs_version("<4.15")
```

Comment:
The skip version doesn't seem to be right. The fix for the BZ 2141422 is in RHCS-6.1.4(4.14.5 and above) and RHCS-5.3.6(4.13.8 and above). Please check again. See: https://bugzilla.redhat.com/show_bug.cgi?id=2141422#c40

Reply:
Fixed In Version: | 4.15.0-123

Ceph version upgraded to 6.1.4  in 4.14, 4.13, 4.12. I am not sure whether it works in all versions of older released or only few z streams of these older versions.